lib/config.js

```diff
@@ -5,4 +5,27 @@
 var config = require('bedrock').config;
 var path = require('path');
 
-config['messages-email'] = {};
+config['messages-email'] = config['messages-email'] ||  {};
+config['messages-email'].emailEventType =
+  config['messages-email'].emailEventType || 'bedrock.messageEmailEventType';
+
+config['messages-email'].defaultUserSettings =
+  config['messages-email'].defaultUserSettings ||
+  {enable: true, interval: 'immediate'};
+
+config.mail.events.push({
+  type: 'bedrock.messageEmailEventType',
+  // email for new message
+  template: 'generic.notification.email'
```

Comment:
We might want to scope the template name to `bedrock-messages-email` as well. We don't really have a convention for this yet -- and it seems like it would be a good idea.